drivers/mpu9250/MPU9250.hpp

```diff
@@ -181,6 +181,9 @@ namespace DriverFramework
 #if defined(__DF_EDISON)
 // update frequency 250 Hz
 #define MPU9250_MEASURE_INTERVAL_US 4000
+#elif defined(__DF_RPI_SINGLE)
+// update frequency 400 Hz
+#define MPU9250_MEASURE_INTERVAL_US 1000
```

Comment:
Hm, I'm confused. Does the comment match the number? And is the define needed? Because it's the same as what's in else.